pkg/build/strategies/sti/sti.go

```diff
@@ -481,8 +500,10 @@ func (b *STI) Execute(command string, user string, config *api.Config) error {
 					return err
 				}
 			}
-			if err := b.docker.UploadToContainer(rmScript, "/tmp/rm-injections", containerID); err != nil {
-				return err
+			if len(user) > 0 {
+				if err := b.docker.ChangeContainerFilesOwner(containerID, user, injectedFiles); err != nil {
```

Comment:
does this have any implication for ssl certs?  not sure if there are any ownership requirements on those.

certainly there are for ssh keys...